index.js

```diff
@@ -232,7 +233,18 @@ async function buildPlatforms () {
 
         if (platformOverride.unprotectedTemporary) {
             addExceptionsToUnprotected(platformOverride.unprotectedTemporary)
-            platformConfig.unprotectedTemporary = platformConfig.unprotectedTemporary.concat(platformOverride.unprotectedTemporary)
+
+            for (const key of Object.keys(platformOverride.features)) {
```

Comment:
This whole function should be able to be shared between the global config code right?

It would be good to share so we don't have implementation drift.